model_compression_toolkit/common/bias_correction/apply_bias_correction_to_graph.py

```diff
@@ -36,7 +36,7 @@ def apply_bias_correction_to_graph(graph_to_apply_bias_correction: Graph,
 
     graph = copy.deepcopy(graph_to_apply_bias_correction)
     for n in graph.nodes:
-        if fw_info.in_kernel_ops(n) and n.final_weights_quantization_cfg.enable_weights_quantization:
+        if n.final_weights_quantization_cfg and n.final_weights_quantization_cfg.enable_weights_quantization:
             # If a kernel was quantized and weights bias correction is enabled in n.quantization_cfg,
```

Comment:
I think it's better to change is_weights_quantization_enabled() to check first if the node has a final configuration, and if so - check this configuration flag. If the node doesn't have a final config yet, keep doing what we're doing today (check candidates).
This way, we can call is_weights_quantization_enabled here as well.